components/webdriver_server/actions.rs

```diff
@@ -788,7 +788,12 @@ impl Handler {
         );
     }
 
+    /// If x or y is less than 0 or greater than the width of the viewport in CSS pixels,
+    /// then return error with error code move target out of bounds.
```

Comment:
Nit:

I think it's great to add documentation here, but it should also be prefixed by a high-level explanation of what the method is meant to do. For instance, you could write something like:

```rust
/// Verify that the given coordinates are within the boundary of the viewport.
/// If x or y is less than 0 or greater than the width of the viewport in CSS pixels,
/// then return error with error code move target out of bounds.
```